scripts/ci/debian/mono-llvm.install

```diff
@@ -0,0 +1,1574 @@
+usr/lib/libLLVMAMDGPUAsmParser.a
```

Comment:
Do we really want to be in /usr? This conflicts with any distribution llvm packages, which might be a problem for people who already use llvm for other things to contribute. Maybe we need our own prefix, /opt/mono-llvm or something?